nix/auto-luks.nix

```diff
@@ -67,6 +67,24 @@ with utils;
               luksFormat</command>.
             '';
           };
+
+          mountPoint = mkOption {
+            default = false;
+            type = types.nullOr types.string;
```

Comment:
"string" is deprecated, it is better "str" or likely "path".

Reply:
Thanks. Changed.